packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -421,6 +436,7 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         if (uri) {
             return uri;
         }
+
```

Comment:
maybe remove this line that do nothing ?

Reply:
i think this is needed so it doesn't continue with 
``` 
const root = this.workspaceService.tryGetRoots()[0];
return root && new URI(root.uri);
```

I can just simplify it with
```
if (this.workspaceService.isMultiRootWorkspaceOpened || uri) {
      return uri;
}
```

Reply:
@uniibu I was just referencing the empty line

Reply:
ah my bad. 👍